operators/multiclusterobservability/bundle/manifests/core.observatorium.io_observatoria.yaml

```diff
@@ -1,3 +1,5 @@
+
+---
```

Comment:
Need to remove the redundant `---`. it will lead to the failure installation.

Reply:
done